src/main/java/xyz/vegaone/security/SecurityApplication.java

```diff
@@ -2,7 +2,9 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.cloud.netflix.eureka.server.EnableEurekaServer;
 
+@EnableEurekaServer
```

Comment:
`@EnableEurekaServer` should be replaced with `@EnableEurekaClient` since the `security` module will register with the Eureka Server which is on the `register` module.